src/platform/site-wide/sass/modules/_m-facilities.scss

```diff
@@ -0,0 +1,57 @@
+// Facilities Hub
+.facility-main-cta {
+  margin: 8px 0;
+}
+
+.facility-main-cta-item {
+  border-left: 4px solid $color-link-default;
+  background-color: $color-primary-alt-lightest;
+  height: 40px;
+  margin-right: 14px;
+  padding-top: 5px;
+  padding-left: 14px;
+  &:last-child {
+    margin-right: 0;
+  }
+  a {
+    color: #3f77b0;
+    text-decoration: none;
+  }
+}
+.facility-hub-cta-item {
+  border-top: 1px solid $color-primary-alt-light;
+  height: 73px;
+  line-height: 73px;
+  position: relative;
+
+  span {
+    display: inline-block;
+    vertical-align: middle;
+    line-height: 20px;
+    margin-left: 48px
+  }
+  a {
+    text-decoration: none;
+  }
+}
+.facility-hub-cta-item-circle {
```

Comment:
Some of the icon centering is off. I've found that I've had to set the top/padding on an icon by icon basis to get that right (the homepage is an example).